src/placement/jump_map.c

```diff
@@ -652,10 +724,10 @@ get_object_layout(struct pl_jump_map *jmap, struct pl_obj_layout *layout,
 
 	dom_used_length = (struct pool_domain *)(root->do_targets) - (root) + 1;
 
-	D_ALLOC_ARRAY(dom_used, dom_used_length);
-	D_ALLOC_ARRAY(used_targets, ((layout->ol_nr) / 8) + 1);
+	D_ALLOC_ARRAY(dom_used, (dom_used_length / 8) + 1);
+	D_INIT_LIST_HEAD(&used_targets_list);
 
-	if (dom_used == NULL || used_targets == NULL)
+	if (dom_used == NULL)
```

Comment:
Not really your bug, but shouldn't this be D_GOTO(out, rc = -DER_NOMEM); ?

Reply:
Thanks for spotting that,
Should be fixed now.